components/permission-controller/README.md

```diff
@@ -0,0 +1,38 @@
+# Permission-controller
+
+## Overview
+Permission-controller provides a mechanism for granting admin privileges within custom Namespaces to selected user groups. Under the hood, the component is a Kubernetes controller that watches for instances of `Namespace core/v1` objects and ensures desired RBAC configuration by creating and updating objects of type `rolebindings.rbac.authorization.k8s.io`.
+
+Click [here](/resources/permission-controller) to access the Helm chart that defines the component's installation.
+
+## Prerequisites
+
+- working Kubernetes cluster
+- go 1.12 or higher
+- docker
+
+## Development
+
+The development process uses formulae declared in the [generic](/common/makefiles/generic-make-go.mk) and [component-specific](./Makefile) Makefiles.
```

Comment:
"formulae" typo? not sure. i'm simple dev ;)

Reply:
@strekm It is by no means a typo. It's just me being linguistic traditionalist (at least as far as English words derived from latin are concerned). The general rule says `formulas` should be used in everyday speech, while `formulae` is limited to scientific contexts. After all, we do computer science, don't we?

Reply:
Only @tomekpapiernik can make me change this 🤣